lib/resource/Account.js

```diff
@@ -94,7 +94,20 @@ Account.prototype.getApiKeys = function getApiKeys(options,callback) {
 
 Account.prototype.save = function saveAccount(){
   var self = this;
-  var args = arguments;
+
+  var args = Array.prototype.slice.call(arguments);
+
+  // If customData, then inject our own callback and invalidate the
+  // customData resource cache when the account finishes saving.
+  if (self.customData) {
+    var originalCallback = args.length > 0 && typeof args[args.length - 1] === 'function' ?
```

Comment:
Yeah, that definitely looks better! :) :+1: 

Also regarding NOP, it's not a typo: https://en.wikipedia.org/wiki/NOP